src/main/java/io/codelex/dateandtime/practice/Exercise1/Exercise1.java

```diff
@@ -0,0 +1,43 @@
+package io.codelex.dateandtime.practice.Exercise1;
+
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+
+public class Exercise1 {
+
+
+    public static void main(String[] args) {
+
+        LocalDate date1 = LocalDate.parse("2022-03-07");
+        LocalDate date2 = LocalDate.parse("2022-03-20");
+
+        // Loop to count days in general
```

Comment:
labi atadlīta loģika, komentāra vietā tik supe rlabi būtu izvilkt uz metodi un iztiktu mez komentāra arī
`private int overallDays()` for example

Reply:
Hey, salabots.